web/src/views/FileBrowserView/FileBrowser.vue

```diff
@@ -370,6 +370,7 @@ const updating = ref(false);
 // Computed
 const owners = computed(() => store.owners?.map((u) => u.username) || null);
 const currentDandiset = computed(() => store.dandiset);
+const embargoed = computed(() => currentDandiset.value && currentDandiset.value.dandiset.embargo_status === 'EMBARGOED');
```

Comment:
This might just be a bit more succinct, as if `currentDandiset` is not set the comparison will be `undefined === 'EMBARGOED'` which will still evaluate to `false`.

```suggestion
const embargoed = computed(() => currentDandiset.value?.dandiset.embargo_status === 'EMBARGOED');
```

Reply:
I think I was maybe satisfying the typechecker? As you point out, the type of the succinct version will be `boolean | undefined`, whereas the longer one is just `boolean`.

Reply:
The succinct version would still return just a `boolean`, as the check of `undefined === 'EMBARGOED'` is `false` (and at least for me locally, results in just a boolean computed ref). In general I'd like to avoid the pattern of `x && x.y` if possible, as I think it's excessively verbose and complicates things.

Reply:
Good analysis, @AlmightyYakob.